creme/proto/robust_soft_learning_vector_quantization.py

```diff
@@ -0,0 +1,388 @@
+import math
+
+import numpy as np
+
+from sklearn.utils import validation
+from sklearn.metrics import euclidean_distances
+from sklearn.utils.multiclass import unique_labels
+
+from creme.base import Classifier
+from creme.utils.skmultiflow_utils import check_random_state
+from creme.utils import dict2numpy
+
+
+class RobustSoftLearningVectorQuantization(Classifier):
```

Comment:
TBH I would call this `RSLVQClassifier`, or `RobustSoftLVQClassifier` considering that `LVQ` is an acronym that is used in research papers. `RobustSoftLearningVectorQuantization` is too big a mouthful IMO :)

I wouldn't worry about using acronyms. Indeed, in the documentation we always show the first line of the docstring to explain the acronym anyway. There's no need to stutter.

Reply:
I renamed the class as `RSLVQClassifier`. It is a good compromise. 

We went from `RSLVQ` -> `RobustSoftLearningVectorQuantization` in an attempt to achieve clarity, but we overdid it :-)